### PR TITLE
SRS Cleanup & Improvements

### PR DESCRIPTION
**Pull request by jGleitz**
Contains minor, yet numerous changes and improvements of the SRS.

### CODE REVIEW

**Comment by flyfl** [comment]
lgtm. But for the next time, I would really appreciate if you could separate the numerous changes you've done, in more than just one commit. Maybe for each file or at least a logical bundle of files that contains similar changes.

**Comment by jGleitz** [comment]
@flyfl you're right, I'm sorry. I'll do this the next time. Thanks for reviewing anyway!

**Comment by RomanLangrehr** [comment]
Good changes, just one little thing:
I think we should write "context menu" (as that is the correct term, see [Wikipedia](https://en.wikipedia.org/wiki/Context_menu) ) rather than context sensitive menu.

**Comment by jGleitz** [comment]
> I think we should write "context menu"

I don’t want to write context menu, as this restricts us to using a context menu. What I’d like to say is that we’ll “show some button at the right spot” – to me, that doesn’t necessarily have to be a context menu.

File: doc/Requirements Specification/Parts/Terms and Definitions.lyx [hunk 1]
```diff
@@ -234,7 +296,7 @@ lstinline{while} loop and the
 \backslash
 lstinline{do}-
 \backslash
-lstinline{while} construct are loops.
+lstinline{while} stements form loops.
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Terms and Definitions.lyx]: typo, I’ll fix it in a minute

**Comment by RomanLangrehr** [comment]
> I don’t want to write context menu, as this would restrict us to using a context menu. What I’d like to say is that we’ll “show some button at the right spot” – to me, that doesn’t necessarily have to be a context menu.

I see. With that point of view it is, of course, a good idea to wirte "context sensitive menu"

**Comment by jGleitz** [comment]
@RomanLangrehr so you’re okay with this now?

**Comment by RomanLangrehr** [comment]
Yes.
Lgtm

**Pull request merged**